packages/e2e-test/src/commands/run.ts

```diff
@@ -66,9 +69,9 @@ export async function run() {
   print('Starting the app');
   await testAppServe(pluginId, appDir);
 
-  if (Boolean(process.env.POSTGRES_USER)) {
-    print('Testing the PostgreSQL backend startup');
-    await preCleanPostgres();
+  if (Boolean(process.env.POSTGRES_USER) || Boolean(process.env.MYSQL_USER)) {
```

Comment:
i guess we need to make up our minds about whether this should be _USER like here, or _CONNECTION like below :)